### ISSUE TITLE
Automatically update dependencies

### ISSUE THREAD

**Issue by infinitewarp** [cloudigrade/cloudigrade#1137]
## Summary

As a cloudigrade dev, I want to stay on top of minor dependency version updates so I don't have to think about and manually update `poetry.lock`.

Before jumping into writing our own script, spend an hour so verifying that no good options already exist as free-to-use bots/GitHub integrations.

## Acceptance Criteria

- [x] Verify an automation action exists somewhere and runs to create new PRs with updated `poetry.lock`
  - Should be easy to verify since we have been manually updating, and there will likely already be new updates available.

## Assumptions and Questions

- How about writing a simple script that is called via a GitHub action? The script could call `poetry lock`, `git commit`, and open a PR with the change.
  - Do we need to use a real GitHub user? If so, grab a token for our @cloudigrade-bot .
- Is pyup still a reliable thing? Does it work with poetry?
  - We probably can't use PyUp because we're not strictly "free" since we are employed by Red Hat and have paying customers.
- What about dependabot?
  - Already active and already creates PRs, but only for critical CVEs, not for routine updates.
- Other services to investigate?
  - ??????

**Comment by infinitewarp** [comment]
See my comment at https://ansible.slack.com/archives/C8VGAPJNN/p1655314927816949

> Since the platform tooling continues to break every time a bot open a PRs, I have suspended the Renovate bot from writing updates to any of our repos. I have forked the main repos into my personal GitHub account and have enabled Renovate to run there instead. This allows me to keep an eye out for new dependency versions without causing problems for the platform's broken CI tool chain. I will try to keep these in sync and periodically open PRs to our repos based on the updates reported in my personal forks.
>
> This is certainly not ideal, but I'd rather have a manual process like this than see ci.int and its friends fall into the void every other day. :confused: Maybe someday later we can have good integrated tools.

**Comment by infinitewarp** [comment]
https://gitlab.cee.redhat.com/service/app-interface/-/merge_requests/42316 has merged! So, we can _probably_ use Revonate bot with our projects now.

**Comment by infinitewarp** [comment]
Renovate bot is back in action, and I've confirmed that it's working in our repos. PR checks are working, and that includes the previously troublesome ci.ext and ci.int jobs. I've accepted and merged several `renovate` PRs since reenabling it (#1315, #1316#, cloudigrade/houndigrade#133, cloudigrade/houndigrade#135), and everything related in our automation appears to be working well.

What do we do going forward?

We just keep an eye out for incoming PRs, review, approve, and merge like any other. Sometimes there will be PRs we want to hold and not merge yet, and that's perfectly fine too.
